### PR TITLE
#835 Added printing available Android software versions

### LINKED ISSUE #835: Show available devices
Hi folks I needed to run for a particular device and couldn't figure out the device name to send. It would be nice to a device lookup that wraps the gcloud command `gcloud firebase test android models list`

Please and thank you

### PR DESCRIPTION
**Pull request by piotradamczyk5**
Fixes #835 

Use on of a new CLI option to test this PR:
- `flank firebase test android versions list` or `flank android versions list` - print Android available versions to test against

<details>
<summary>Sample Android versions list output:</summary>
<p>

```
┌───────────────┬─────────┬─────────────┬───────────┬──────────────┬─────────┐
│ OS_VERSION_ID │ VERSION │  CODE_NAME  │ API_LEVEL │ RELEASE_DATE │  TAGS   │
├───────────────┼─────────┼─────────────┼───────────┼──────────────┼─────────┤
│      18       │  4.3.x  │ Jelly Bean  │    18     │  2013-10-03  │         │
│      19       │  4.4.x  │   KitKat    │    19     │  2014-06-02  │         │
│      21       │  5.0.x  │  Lollipop   │    21     │  2014-12-19  │         │
│      22       │  5.1.x  │  Lollipop   │    22     │  2015-04-21  │         │
│      23       │  6.0.x  │ Marshmallow │    23     │  2015-10-05  │         │
│      24       │  7.0.x  │   Nougat    │    24     │  2016-08-22  │         │
│      25       │  7.1.x  │   Nougat    │    25     │  2016-10-19  │         │
│      26       │  8.0.x  │    Oreo     │    26     │  2017-08-21  │ default │
│      27       │  8.1.x  │  Oreo MR1   │    27     │  2017-12-04  │         │
│      28       │   9.x   │     Pie     │    28     │  2018-08-06  │         │
│      29       │  10.x   │      Q      │    29     │  2019-09-03  │         │
│      30       │   11    │      R      │    30     │  2020-09-03  │         │
└───────────────┴─────────┴─────────────┴───────────┴──────────────┴─────────┘

```
</details>

More listing options of the test environment will be added in the next tasks

## Checklist

- [x] printing Android software versions list
- [x] Documented
- [x] Unit tested
- [x] release_notes.md updated

### CODE REVIEW

File: test_runner/src/test/kotlin/ftl/cli/firebase/test/android/versions/AndroidVersionsListCommandTest.kt [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import com.google.common.truth.Truth.assertThat
+import ftl.config.FtlConstants
+import org.junit.Test
+import picocli.CommandLine
+
+class AndroidVersionsListCommandTest {
+
+    @Test
+    fun androidVersionsListCommandOptions() {
```
Comment (adamfilipow92) [inline test_runner/src/test/kotlin/ftl/cli/firebase/test/android/versions/AndroidVersionsListCommandTest.kt:11]: What are the purposes of this test?

Comment (piotradamczyk5) [reply test_runner/src/test/kotlin/ftl/cli/firebase/test/android/versions/AndroidVersionsListCommandTest.kt:11]: checked if parameters are set

Comment (jan-gogo) [reply test_runner/src/test/kotlin/ftl/cli/firebase/test/android/versions/AndroidVersionsListCommandTest.kt:11]: There is no logic under the hood. Why do that?

Comment (piotradamczyk5) [reply test_runner/src/test/kotlin/ftl/cli/firebase/test/android/versions/AndroidVersionsListCommandTest.kt:11]: just follow testing practices from other `Command`s
I will remove it

Comment (jan-gogo) [reply test_runner/src/test/kotlin/ftl/cli/firebase/test/android/versions/AndroidVersionsListCommandTest.kt:11]: Yup, I see. Now I found that we should change those practices. There is no reason for testing assignments.

Comment (piotradamczyk5) [reply test_runner/src/test/kotlin/ftl/cli/firebase/test/android/versions/AndroidVersionsListCommandTest.kt]: Fixed

**Pull request merged**